python/port/mod/ppn/modppn_table.c

```diff
@@ -0,0 +1,23 @@
+#include "modppn.h"
+
+STATIC MP_DEFINE_CONST_FUN_OBJ_1(modppn_colLst2Int_obj, modppn_colLst2Int);
+STATIC MP_DEFINE_CONST_FUN_OBJ_1(modppn_colInt2Lst_obj, modppn_colInt2Lst);
+
+STATIC const mp_rom_map_elem_t modppn_module_globals_table[] = {
+  { MP_ROM_QSTR(MP_QSTR___name__), MP_ROM_QSTR(MP_QSTR_ppn) },
+  { MP_ROM_QSTR(MP_QSTR_ppn_w), MP_OBJ_NEW_SMALL_INT(screenWidth)},
+  { MP_ROM_QSTR(MP_QSTR_ppn_h), MP_OBJ_NEW_SMALL_INT(screenHeight)},
+  { MP_ROM_QSTR(MP_QSTR_ppn_y0), MP_OBJ_NEW_SMALL_INT(screenY0)},
+  { MP_ROM_QSTR(MP_QSTR_ppn_has_color), hasColor ? mp_const_true : mp_const_false},
```

Comment:
If this library only run on numworks, why having ternary condition, which will take useless ROM

Reply:
Globaly, try to use more precompilator directives to avoid useless ROM usage

Reply:
I did this change in local because, I think it is the cause of the error of compilation.